### DISCUSSION TITLE
Allow "Hide Top Toolbar" to be enabled separately

### DISCUSSION THREAD

**Discussion by mr-cheffy** [zen-browser/desktop#879] [Ideas]
### What feature would you like to see?

Would love to be able to hide the top toolbar without having full compact mode enabled. I like being able to see my tabs, but want to hide the address bar.

![Screenshot from 2024-08-11 16-45-25](https://github.com/user-attachments/assets/5ec9d215-4065-49e2-a3f4-3ce3bebad7b6)

### What platform are you seeing the problem on?

Linux

 
 ~ _Created automatically from issue: https://github.com/zen-browser/desktop/issues/229_

**Comment by migtorr** [comment]
A theme has been made for that now https://www.zen-browser.app/themes/d8a9b9a4-30b7-48b5-99ad-448efa604f7e
